src/main/java/com/direwolf20/justdirethings/client/blockentityrenders/baseber/GooBlockRender_Base.java

```diff
@@ -73,16 +75,12 @@ public void render(T blockentity, float partialTicks, PoseStack matrixStackIn, M
     }
 
     private ItemStack getNextItemFromTag(int tier) {
-        TagKey<Item> tag = switch (tier) {
-            case 1 -> JustDireItemTags.GOO_REVIVE_TIER_1;
-            case 2 -> JustDireItemTags.GOO_REVIVE_TIER_2;
-            case 3 -> JustDireItemTags.GOO_REVIVE_TIER_3;
-            case 4 -> JustDireItemTags.GOO_REVIVE_TIER_4;
-            default -> null;  // Handle unexpected tiers
-        };
-        if (tag == null)
-            return ItemStack.EMPTY;
-        List<Holder<Item>> items = BuiltInRegistries.ITEM.getTag(tag).stream().flatMap(h -> h.stream()).toList();
+
+        List<Holder<Item>> items = BuiltInRegistries.ITEM
+                .getTag(TagKey.create(Registries.ITEM,
```

Comment:
I don't like the dynamic creation of these tags, I feel like if we know the level of tiers, we could precompute them? If not, maybe a lazy list or something so we're not recreating it each time?